framework/include/material_image_helpers.hpp

```diff
@@ -88,80 +103,175 @@ namespace gvk
 	}
 
-	// create cubemap from image data
+	/** Create cubemap from image data, with optional caching
+	* @param aImageData		an valid instance of image_data containing cubemap image data.
+	* @param aMemoryUsage	the intended memory usage of the returned image resource.
+	* @param aImageUsage	the intended image usage of the returned image resource.
+	* @param aSyncHandler	the sync handler to use creating the image resource.
+	* @param aSerializer	a serializer to use for caching data loaded from disk. 
+	*/
 	avk::image create_cubemap_from_image_data_cached(image_data& aImageData, avk::memory_usage aMemoryUsage = avk::memory_usage::device,
 		avk::image_usage aImageUsage = avk::image_usage::general_cube_map_texture, avk::sync aSyncHandler = avk::sync::wait_idle(), std::optional<std::reference_wrapper<gvk::serializer>> aSerializer = {});
 
+	/** Create cubemap from image data, with caching
+	* @param aSerializer	a serializer to use for caching data loaded from disk. 
+	* @param aImageData		an valid instance of image_data containing cubemap image data.
+	* @param aMemoryUsage	the intended memory usage of the returned image resource.
+	* @param aImageUsage	the intended image usage of the returned image resource.
+	* @param aSyncHandler	the sync handler to use creating the image resource.
+	*/
 	static avk::image create_cubemap_from_image_data_cached(gvk::serializer& aSerializer, image_data& aImageData, avk::memory_usage aMemoryUsage = avk::memory_usage::device, 
 		avk::image_usage aImageUsage = avk::image_usage::general_cube_map_texture, avk::sync aSyncHandler = avk::sync::wait_idle())
 	{
 		return create_cubemap_from_image_data_cached(aImageData, aMemoryUsage, aImageUsage, std::move(aSyncHandler), aSerializer);
 	}
 
+	/** Create cubemap from image data
+	* @param aImageData		an valid instance of image_data containing cubemap image data.
+	* @param aMemoryUsage	the intended memory usage of the returned image resource.
+	* @param aImageUsage	the intended image usage of the returned image resource.
+	* @param aSyncHandler	the sync handler to use creating the image resource.
+	*/
 	static avk::image create_cubemap_from_image_data(image_data& aImageData, avk::memory_usage aMemoryUsage = avk::memory_usage::device,
 		avk::image_usage aImageUsage = avk::image_usage::general_cube_map_texture, avk::sync aSyncHandler = avk::sync::wait_idle())
 	{
 		return create_cubemap_from_image_data_cached(aImageData, aMemoryUsage, aImageUsage, std::move(aSyncHandler));
 	}
 
-	// create cubemap from a single file
+	/* create cubemap from a single file, with optional caching
+	* @param aPath			file name of a texture file to load the image data from.
+	* @param aMemoryUsage	the intended memory usage of the returned image resource.
+	* @param aImageUsage	the intended image usage of the returned image resource.
+	* @param aSyncHandler	the sync handler to use creating the image resource.
+	* @param aSerializer	a serializer to use for caching data loaded from disk.
```

Comment:
These parameter descriptions do not match the method's actual parameters.

Reply:
Please start parameter descriptions with a capital letter!! Especially if you end the description with a period!

Reply:
This actually applies to almost all parameter descriptions.

Reply:
Hmm, probably not worth the effort.. but it would have been nice. Please change it if you find an easy way to change it... otherwise please pay attention to it in the future!